crates/store/re_types_core/src/reflection.rs

```diff
@@ -253,12 +258,15 @@ impl ArchetypeReflection {
 /// Additional information about an archetype's field.
 #[derive(Clone, Debug)]
 pub struct ArchetypeFieldReflection {
-    /// The type of the field (it's always a component).
-    pub component_name: ComponentName,
+    /// The name of the field (i.e. same as `ComponentDescriptor::archetype_field_name`).
+    pub name: &'static str,
```

Comment:
Mandatory in order to be able to recreate complete `ComponentDescriptors` at runtime.